src/js/letters/actions/letters.js

```diff
@@ -236,13 +228,19 @@ export function saveAddress(address) {
   };
   window.dataLayer.push({ event: 'letter-update-address-submit' });
   return (dispatch) => {
-    // TODO: Show a spinner or some kind of indication we're waiting on this to return
     dispatch(saveAddressPending());
-
     return apiRequest(
       '/v0/address',
       settings,
-      () => dispatch(saveAddressSuccess(address)),
+      (response) => {
+        // translate military address properties back to front end address
+        const responseAddress = militaryToGeneric(response.data.attributes.address);
+        if (!isEqual(stripEmpties(address), stripEmpties(responseAddress))) {
+          const mismatchError = new Error('letters-address-update addresses don\'t match');
+          Raven.captureException(mismatchError, { submitted: address, returned: responseAddress });
```

Comment:
Let's make sure we can log this before we merge it in. Not sure if it counts as PII.

Reply:
Oh, that's a great call...I'll check into it

Reply:
Checked with @kreek. Seems as though you could reverse engineer PII from the address itself so changed the code to ensure we don't log any of the actual address information itself.